espanol/pensamiento/estoicismo/estoicismo.md

```diff
@@ -1,3 +1,5 @@
 La reflexi�n iba arrastrandose por los rincones de su subconciente queriendo hacerse visible. Tras hacerse paso entre 
 el caos provocado por sus lamentos, inund� su mente. Esto es lo que, de manera simplista, pens� el Consejero:
-"Las cosas son como son, no como quiero que sean. �Por qu� sufrir?, debo aceptar la realidad y seguir adelante."
\ No newline at end of file
+"Las cosas son como son, no como quiero que sean. �Por qu� sufrir?, debo aceptar la realidad y seguir adelante."
+
+Y asi lo transmitio a las personas que esperaban de sus sabios consejos. De esta menera los lleno de coraje para enfrentar lo que venia de manera inevitable.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.